.github/workflows/enforce-label.yml

```
@@ -5,6 +5,11 @@ permissions: {}
 on:
   pull_request:
     types: [labeled, unlabeled, opened, edited, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
```

Why do we want this? I understand why cancelling deployment when a new update is pushed makes sense, but here I worry that we could end up cancelling something when one changes labels. Since this action runs in seconds I am not sure if it is worth it, but if you believe it is I would add a group differentiator like:


```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}-enforce-label
  cancel-in-progress: true
```